bbl_screen-patch/kexec_ui/printerui/qml/dialog/KexecDialog.qml

```diff
@@ -84,10 +133,6 @@ Item {
         font: Fonts.body_32
         color: Colors.gray_200
         wrapMode: Text.Wrap
-        text: hasSdCard 
-            ? (countdown > 0) 
-                ? qsTr("Your printer will automatically boot from the SD card in %1 seconds.").arg(countdown)
-                : qsTr("Your printer is rebooting into the OS on the inserted SD card.")
-            : qsTr("Insert a bootable SD card and restart the printer, or use the startup options menu to repair your X1Plus installation.")
+        text: hasSdCard ? startupStrings[startupMode].subtitle() : qsTr("Please insert an SD card to continue.")
```

Comment:
did you test this?  inserting a SD card will not clear this flag, as written, and this can happen if there is a SD card but it does not contain a boot.conf.

Reply:
I will change it back to its original caption. New strings here = more translations so there was no point in changing to begin with.